_posts/2019-09-23-kikcms.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: post
+title: 'Introducing KikCMS'
+date: 2019-09-23
+tags:
+  - CMS
+  - phalcon
+  - UX
+  - MySQL
+---
+
+# KikCMS
+
+Since 2017, I started being self-employed full time. I build websites and web applications for companies. Every project has been build upon the KikCMS, which is build upon the Phalcon framework.
+
+How and why it started
+I’ve worked for an agency from 2005 until 2015. I was the lead developer of the CMS they are using. The main reasons we chose to build our own was to be more flexible. We wanted to have a CMS that could do anything (and was reasonable) the client asked for. 
+
+I started the development of their CMS in 2006. It had two major overhauls and the version they still use today stems from 2010. Also the design changed quite a bit in 2015, but technically and functionally didn’t change much. 
+
+A few years before I left the company, I really wanted to build a brand new version. The current version was build without a framework, and that caused a lot of problems and hurt productivity. Apart from that I knew that I couldn’t develop the CMS the way I envisioned it to be, with multiple people having a say in it.
+
+So, after working 1,5 years at another company tweakers, I decided to quit my job, and start my own company. I had this idea in my mind of building websites myself for my own clients in my own build CMS. I knew this was a tall order to fill, but I was bursting with so much motivation that I just felt like I had to do it.
+
+## The goal
+So in 2017 I started for myself. By some luck, I quickly found a few projects I could work on. But because I didn’t started working on the CMS yet, I had to use an older version of the CMS I’ve build for the company which I forked years ago to do some of my own projects. It looked like this:
+
+![KikCMS old version](/assets/files/kikcms-old.jpg "KikCMS old version")
+
+As you can see, I already named it KikCMS. It did the job it had to do, and it looked nice on the outside, but the code was a mess.
```

Comment:
As you can see, I had already named it KikCMS. It did was it was supposed to do, and although it looked nice on the outside, the code was a mess